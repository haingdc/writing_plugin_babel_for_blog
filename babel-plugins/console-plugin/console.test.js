const babel = require('@babel/core')
const plugin = require('./console') // Đường dẫn đến plugin của bạn

/** @type {import('@babel/core').TransformOptions} */
const opts = {
  plugins: [plugin],
  presets: ['@babel/preset-env'],
}

describe('tại global scope', () => {
  it('đơn giản', () => {
    const code = `inspect('hello babel');`
    const output = babel.transformSync(code, opts)
    expect(output.code).toMatchSnapshot()
  })
})
