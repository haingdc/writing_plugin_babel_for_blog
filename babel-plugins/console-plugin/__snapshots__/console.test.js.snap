// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tại global scope đơn giản 1`] = `
""use strict";

inspect('hello babel');"
`;
